resources/test-data.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Swedbank Pay Resources – Introduction
+opengraph:
+    description: Introduction to Swedbank Pay Resources
+sidebar:
+  navigation:
+  - title: Checkout
```

Comment:
I don't think Checkout's menu should be on the `resource` pages. :)

Reply:
Agree! It's unnecessary to have that menu in the resources section. Removed